lib/src/components/supa_socials_auth.dart

```diff
@@ -54,8 +58,13 @@ extension on OAuthProvider {
         _ => Colors.black,
       };
 
-  String get labelText =>
-      'Continue with ${name[0].toUpperCase()}${name.substring(1)}';
+  String get capitalizedName {
+    String modifiedName = name;
+    if (name.endsWith("Oidc")) {
+      modifiedName = name.substring(0, name.length - 4);
+    }
+    return modifiedName[0].toUpperCase() + modifiedName.substring(1);
+  }
```

Comment:
Let's keep the same `labelText` getter name, but modify the definition in a way that the `Oidc` is removed.